clang/lib/Format/FormatToken.h

```diff
@@ -1850,6 +1831,7 @@ struct AdditionalKeywords {
   bool isVerilogBegin(const FormatToken &Tok) const {
     // `table` is not included since it needs to be treated specially.
     return !Tok.endsSequence(kw_fork, kw_disable) &&
+           !Tok.endsSequence(kw_fork, kw_wait) &&
            Tok.isOneOf(kw_begin, kw_fork, kw_generate, kw_specify);
```

Comment:
This can be made more efficient and readable, e.g.:
```suggestion
    if (Tok.isOneOf(kw_begin, kw_generate, kw_specify))
      return true;
    if (Tok.is(kw_fork)) {
      const auto *Prev = Tok.getPreviousNonComment();
      return !(Prev && Prev->isOneOf(kw_disable, kw_wait));
    }
```